app/services/chat_service.rb

```diff
@@ -10,40 +15,48 @@ def self.get_story(winner, loser)
         },
         {
           role: "user",
-          content: "#{winner} beat #{loser} in a fight.  Please give a short story of how the battle went."
+          content: user_content
         }
-      ]
-    }.to_json
-    get_url('/v1/chat/completions', payload)
+      ],
+      temperature: 0.8,
+      max_tokens: 600,
+      top_p: 1,
```

Comment:
nice add.  if we have to come and adjust for specifics for chatgpt we may need to revisit this.